dataland-frontend/src/utils/GenericFrameworkTypes.ts

```diff
@@ -1,7 +1,7 @@
 import { type DropdownOption } from "@/utils/PremadeDropdownDatasets";
 import {
   type EuTaxonomyDataForFinancials,
-  type EuTaxonomyDataForNonFinancials,
+  type EuTaxonomyDataForNonFinancials, GdvData,
```

Comment:
Why are those in the same line? Was this lintered?

Reply:
They are no longer. Linter probaply fixed it.